.circleci/config.yml

```diff
@@ -58,7 +58,10 @@ commands:
           command: cortex cluster up << parameters.config >> --configure-env aws -y
       - run:
           name: Run E2E Tests
-          command: pytest -v test/e2e/tests --env aws
+          command: |
+            pytest -v test/e2e/tests --env aws --skip-autoscaling --skip-load --skip-long-running
+            pytest -v test/e2e/tests --env aws -k test_autoscaling
+            pytest -v test/e2e/tests --env aws -k test_load
```

Comment:
this can be summed up to a single command

Reply:
That would be ideal - and it is possible, but we'd have to create dependencies (within pytest) between the single-request tests and the load/autoscaling tests. There's no point in letting a load test run if the single-request test failed. I saw this plugin https://pytest-dependency.readthedocs.io/en/stable/usage.html that we could use, but from what I read, this only works for tests that sit in the same module.

The simplest next thing is to just create these "dependencies" by separating the pytest command into multiple ones.